Example/Controllers/Settings/UserSettingsTableViewController.swift

```diff
@@ -35,6 +35,7 @@ class UserSettingsTableViewController: RATTableViewController, UIImagePickerCont
             return
         }
         editPhotoButton.setTitle(editTitleText, for: .normal)
+        deletePhotoButton.setTitle("Remove Photo", for: .normal)
```

Comment:
We should not forget to call localizations extensions for Strings (the suggestion might be ok, but you might want to create a proper key):

```suggestion
        deletePhotoButton.setTitle(MASDKLocale.localize("Remove Photo"), for: .normal)
```